x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/customization_callout.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { EuiButton, EuiFlexGroup, EuiIcon, EuiSpacer, EuiText } from '@elastic/eui';
+
+interface Props {
+  onClick(): void;
+}
+
+export const CustomizationCallout: React.FC<Props> = ({ onClick }) => {
+  return (
+    <EuiFlexGroup
+      direction="column"
+      className="customization-callout"
```

Comment:
We should be camelCasing CSS classes to match EUI/Kibana as a reminder

Also we didn't appear to actually import any custom CSS with this: we should probably do so

```scss
.customizationCallout
  background-color: $euiPageBackgroundColor;
  padding: $euiSizeL;
}
```

Reply:
Ah just saw the CSS class was removed in a later commit. IMO we should still port over the styling, I don't see a strong reason to drop it?

Reply:
I didn't realize there were custom styles before. I can port them over.

Reply:
8160382a4d1b340df4982b3f811af7690943189a